recipe/migrations/python312.yaml

```diff
@@ -33,6 +33,8 @@ python:
   - 3.12.* *_cpython
 # additional entries to add for zip_keys
 numpy:
-  - 1.26
+  - 2.0
```

Comment:
@h-vetinari This will break all feedstocks with Python 3.12. Please revert.

Cc @hmaarrfk

Reply:
How so? I've tested this with a bunch of feedstocks building for 3.12 successfully.

I'm happy to revert of course, but can someone describe (or link to) the breakage?

Reply:
Ah nevermind, I understand what you mean. It changes py312 builds on feedstocks unrelated to numpy. 😑

Reply:
That's an interesting wrinkle. I'm not sure how we can then do the numpy migration for 3.12...

Reply:
Done: https://github.com/conda-forge/conda-forge-pinning-feedstock/commit/09f6263c8dfebfaaf125c0e78dd694ab6e012093

Reply:
I've continued the discussion how to fix this in https://github.com/conda-forge/conda-forge.github.io/issues/1997